docs/scenarios_authoring/Spel.md

```diff
@@ -190,23 +190,20 @@ To obtain the last matching element, the syntax is `.$`.
 ### Transforming lists
 
 Special variable `#this` is used to operate on single element of list.
-            
-Examples below assume following structure:
```

Comment:
To me this is confusing, if not completely wrong. We do not have an assignment ('=') operator in SpEL; we should not suggest such exists. The references to person1, person2 are incorrect. Finally the toMap() method requires some explanation.